### PR TITLE
Masterbar: Fix icon overlap issue at smaller resolutions

### PR DESCRIPTION
**Pull request by lupus2k**
Related to https://github.com/Automattic/dotcom-forge/issues/8378

## Proposed changes:
Fix icon overlap issue at smaller resolutions matching [Calypso fix](https://github.com/Automattic/wp-calypso/pull/92967)
- Below 480px, I reduced the width of icons in masterbar down to 46px to make more space
- Below 480px I hid the "+" icon
- Below 360px I hide the reader icon

> [!IMPORTANT]  
> Some icons on the right panel could show in different places (related to https://github.com/Automattic/dotcom-forge/issues/8443)

### Desktop
| | |
| --- | --- |
| WP-Admin | image |
| Calypso | image |

### <780px
| | |
| --- | --- |
| WP-Admin | image |
| Calypso | image |

### <480px
| | |
| --- | --- |
| WP-Admin | image |
| Calypso | image |

### <360px
| | |
| --- | --- |
| WP-Admin | image |
| Calypso | image |

## Jetpack product discussion
- 

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Apply this PR using [the instructions in the first comment](https://github.com/Automattic/jetpack/pull/38551#issuecomment-2251367129).
* Go to `/wp-admin`
* Check the icon sizes and positions in the selected resolutions.

### CODE REVIEW

**Review by okmttdhr — DISMISSED** [review]

File: projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss [hunk 1]
```diff
@@ -143,6 +172,28 @@
 				}
 			}
 		}
+		@media (max-width: 480px) {
+			#wp-admin-bar-help-center {
+				width: 46px !important;
+				.ab-item {
+					svg {
+						width: 30px;
```
Comment (fushar) [inline projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss]: Why do we need this? In my testing this makes the help center weirdly smaller than the other icons, which do not get reduced in size in this resolution.

**Review by fushar — COMMENTED** [review]
I found that at certain resolution, e.g. 950px, the help center disappears; can we check?

image

File: projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss (hunk 1 above)
Comment (lupus2k) [reply projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss]: Before [this one](https://github.com/Automattic/wp-calypso/pull/92988), the icon was smaller when screen <480px. I'll remove this to make it consistent with other icons.

**Review by fushar — COMMENTED** [review]
On site frontend, the left margin of the `W` logo is too small now for <480px resolution:

image

**Comment by fushar** [comment]
I pushed a small refactor (https://github.com/Automattic/jetpack/pull/38551/commits/ff79eaba31d112d67cb22b8fc0ce9d524b32caf6) but didn't fix the issues above.

Remaining issues:

- Help center disappearing on < 960px resolution (but might not be related to this PR?)
- `W` logo has too small left margin on site frontend

**Comment by fushar** [comment]
Fixed the site frontend issue; now the W logo is placed correctly:

image

**Review by fushar — APPROVED** [review]

**Pull request merged**